### PR TITLE
feat: add coinbase login

### LINKED ISSUE #328: Add coinbase login
@mikeghen assign it to me 🤗

### PR DESCRIPTION
**Pull request by bigint**
https://user-images.githubusercontent.com/69431456/152798857-9eda6a59-c2e3-4441-b51c-14b788070b19.mov

Closes #328

### CODE REVIEW

File: src/store/main/sagas/mainCheckSaga.ts [hunk 1]
```diff
@@ -23,6 +25,28 @@ export function* mainCheckSaga(payload: { init:boolean }) {
         },
       },
     },
+    'custom-coinbase': {
```
Comment (msoni89) [inline src/store/main/sagas/mainCheckSaga.ts:28]: Why Custom-coinbase? Web3Modal support walletlink

Comment (msoni89) [reply src/store/main/sagas/mainCheckSaga.ts:28]: I have not tested it but can you confirm is it support mobile app?

Comment (bigint) [reply src/store/main/sagas/mainCheckSaga.ts:28]: In web3 modal we don't have coinbase support, can u give some examples where u saw it 🙏

Comment (msoni89) [reply src/store/main/sagas/mainCheckSaga.ts:28]: > In web3 modal we don't have coinbase support, can u give some examples where u saw it 🙏

https://github.com/Web3Modal/web3modal/releases coinbase support

Comment (bigint) [reply src/store/main/sagas/mainCheckSaga.ts]: Woah awesome, removed the custome code @msoni89, thanks 🙏🏻

**Review by mikeghen — APPROVED** [review]
Great feature! I didn't know coinbase had a browser extension

**Pull request merged**